docs/dev-setup.rst

```diff
@@ -90,7 +96,21 @@ To setup your local environment you should create a virtualenv and install the
 necessary requirements::
 
     mkvirtualenv --python=/usr/bin/python3.4 serviceinfo
```

Comment:
Well, I don't think we should need to include documentation for how to use one particular tool for installing Python on one platform in the ServiceInfo docs.

How about if we just write:

    mkvirtualenv -p <path>/python3.4 serviceinfo

That leaves it to the developer to know where their python3.4 is, but presumably they know that or they're not going to get very far with Django development.